manifests/Jimmys-Ship-Emporium.yaml

```
@@ -0,0 +1,14 @@
+name: Jimmys-Ship-Emporium
+authors: Jimmy Firyh
+homepage: https://github.com/JimmyFiryh/Jimmys-Ship-Emporium
+license: CC-BY-SA-4.0
+version: v0.1.1
+shortDescription: Simply adds the Glaive, a spicy Republic Navy ship produced after the Kor Sestor incursion.
```

The shortDescription is generally supposed to be a bit shorter. How about this:
```suggestion
shortDescription: Adds the Navy Glaive.
```